src/gov/nasa/worldwind/render/SurfaceText.java

```diff
@@ -42,6 +42,8 @@
     protected CharSequence text;
     /** Location at which to draw the text. */
     protected Position location;
+    /** The angle of text rotation from the true north (clockwise). */
+    protected double rotationAngle;
```

Comment:
Can you change the name to heading? You'll see this is the name used in `SurfaceQuad`. Can you also change the type to `Angle`? This is the default type that is used to represent angles in WorldWind. You can initialize it as:

    private Angle heading = Angle.ZERO;